Kernel/Kernel/Stack.c

```diff
@@ -0,0 +1,39 @@
+#include <Kernel/Stack.h>
+#include <LibK/stdio.h>
+#include <stdint.h>
+
+#if defined(__x86_64__)
+void dumpStack(void) {
+    kprintf("Stacktrace:\n");
+    uint64_t rbp = 0;
+
+    __asm__ volatile("mov %%rbp, %0" : "=r"(rbp));
+
+    for (uint64_t frame = 0; rbp && frame < DUMP_STACK_ENTRIES_MAX; ++frame) {
+        kprintf(" [%p]\n", rbp);
+        rbp += sizeof(void *);
+    }
+}
+
+#elif defined(__i386__)
```

Comment:
We can drop the i386 code completely as I don't plan on porting blobOS to this architecture.

The if macro is a good idea tho, it'll be helpful when we add support for other architecture :)

Reply:
I see. That code hasn't been tested so it's safe to remove